packages/@okta/vuepress-site/docs/guides/pwd-optional-widget-sign-in-email/main/aspnet/integrationsummary.md

```diff
@@ -2,4 +2,8 @@
 
 ![Sequence diagram illustrating the Sign-In Widget's password-optional sign-in use case.](/img/pwd-optional/pwd-optional-dotnet-widget-sign-in-summary.png)
```

Comment:
1. key icon missing in last step.
2. Sphere shouild be diamond on the left of the big blue box.

Reply:
1. not fixing
2. It could be either but as it's on a bidirectional line with circles, it is still a circle.